shared/fixes/bash/display_login_attempts.sh

```diff
@@ -1,14 +1,10 @@
 # platform = Red Hat Enterprise Linux 7
 
-if ! `grep -q ^[^#].*pam_succeed_if.*showfailed /etc/pam.d/postlogin` ; then
-  if ! grep `^session.*pam_succeed_if.so /etc/pam.d/postlogin` ; then
-    echo "session     [default=1]   pam_lastlog.so nowtmp showfailed" >> /etc/pam.d/postlogin
-    echo "session     optional      pam_lastlog.so silent noupdate showfailed" >> /etc/pam.d/postlogin
-  else
-    sed -i --follow-symlinks '/^session.*pam_succeed_if.so/a session\t    optional\t  pam_lastlog.so silent noupdate showfailed' /etc/pam.d/postlogin
-    sed -i --follow-symlinks '/^session.*pam_succeed_if.so/a session\t    [default=1]\t  pam_lastlog.so nowtmp showfailed' /etc/pam.d/postlogin
-  fi
+if ! `grep -q "^session.*pam_lastlog.so" /etc/pam.d/postlogin` ; then
+	echo "session     [default=1]   pam_lastlog.so nowtmp showfailed" >> /etc/pam.d/postlogin
+	echo "session     optional      pam_lastlog.so silent noupdate showfailed" >> /etc/pam.d/postlogin
 else
-  sed -i --follow-symlinks "s/session[ ]*\[default=1][ ]*pam_lastlog.so.*/session     [default=1]   pam_lastlog.so nowtmp showfailed/g" /etc/pam.d/postlogin
-  sed -i --follow-symlinks "s/session[ ]*optional[ ]*pam_lastlog.so.*/session     optional      pam_lastlog.so silent noupdate showfailed/g" /etc/pam.d/postlogin
+	sed -i --follow-symlinks "/pam_lastlog.so/d" /etc/pam.d/postlogin
+	echo "session     [default=1]   pam_lastlog.so nowtmp showfailed" >> /etc/pam.d/postlogin
+        echo "session     optional      pam_lastlog.so silent noupdate showfailed" >> /etc/pam.d/postlogin
```

Comment:
This line is space-indented, unlike others that have tabs.
Moreover, those two lines are identical, so you can consider defining a function that appends those two lines to `postlogin`.
Either way, you can have one `if` with sed delete (i.e. without `else`), and just append those lines after that `if` block.